jupyter/nmm-game-utils.ipynb

```diff
@@ -49,16 +37,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Calculates the opponent of the given player\n",
-    "def opponent(p):\n",
-    "    return 'b' if p == 'w' else 'w'"
+    "START_STATE = ((9, 9), (\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '),\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' '),\n",
+    "    (' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ')\n",
+    "))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Stones"
+    "## Funktionen um mit Steinen zu interagieren"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### hasPlaceableStones\n",
```

Comment:
Der Name allein sieht etwas verloren aus, was hälst du davon die Signatur zu verwenden?

`hasPlaceableStones(State, Player): boolean`

Reply:
Hmm hab ich in gui auch so gemacht und da wurde es nicht requested

Dann müsste ich ja alles ändern 😭😂

Reply:
Ja good point. Mit der Lösung bin ich auch nicht wirklich zufrieden 😅
Brauchen wir überhaupt die Überschriften? Wir haben doch schon die größeren

Reply:
Entfernt